### PR TITLE
mostly working navbar

### PR DESCRIPTION
**Pull request by emilyslouie**
some of the stuff from this [ticket](https://www.notion.so/uwblueprintexecs/Navbar-polish-b631fd0a1c334b87b77b84a9fb37f0e4)

wasn't able to do the drop shadow or the arrow hover or the font sizes since css is killing me :'(

### CODE REVIEW

**Review by GodGreg — APPROVED** [review]
LGTM. Tested.

File: src/Components/Header/Header.jsx [hunk 1]
```diff
@@ -151,49 +156,71 @@ function Header({ program, loadProgram, history, admin, curRoute, routes }) {
             </div>
           </Tooltip>
         </AppName>
-        <p>
-          {" • "}
```
Comment (ahmedhamodi) [inline src/Components/Header/Header.jsx:155]: No more dot. Intentional?

**Review by ahmedhamodi — APPROVED** [review]
Code looks good to me. Small comment on the removal of the dot - is this intentional?

File: src/Components/Header/Header.jsx (hunk 1 above)
Comment (emilyslouie) [reply src/Components/Header/Header.jsx:155]: Oh, I thought the dot might be getting replaced with a coloured dot with the customization stuff. Wasn't sure though.

Comment (ahmedhamodi) [reply src/Components/Header/Header.jsx:155]: Oh interesting, perhaps. I thought the coloured dot was just for form customization, not the navbar too. @GodGreg or @brittanylau, can you comment on the expected look here?

Comment (GodGreg) [reply src/Components/Header/Header.jsx:155]: Designs have a dot. Brittany had mentioned that we can exclude it.

**Pull request merged**